cluster/images/provider-terraform/Dockerfile

```diff
@@ -1,15 +1,32 @@
-FROM hashicorp/terraform:1.2.3
+FROM alpine:3.16.2
+RUN apk --no-cache add ca-certificates bash
 
 ARG TARGETOS
 ARG TARGETARCH
 
-ADD bin/$TARGETOS\_$TARGETARCH/provider /usr/local/bin/crossplane-terraform-provider
+ENV TERRAFORM_VERSION=1.3.4
+ENV TF_CLI_CONFIG_FILE /terraform/.terraformrc
+ENV TF_FORK 0
```

Comment:
Could you document the usage of this variable?

Reply:
it is useful when debugging the provider, prevents terraform from forking if it panics or crashes. I have added a comment to explain this